### PR TITLE
Add Python bindings for CMSSW

### PR DESCRIPTION
**Pull request by cvuosalo**
Add Python bindings for CMSSW to address issue #37. The patch from https://github.com/cms-externals/frontier_client/pull/2 is included.

The following test suggested by Shahzad was completed successfully:

- #create CMSSW_13_0_X dev area
- cmsenv
- export LD_LIBRARY_PATH=/you/frontier_client/lib:${LD_LIBRARY_PATH}
- export PYTHON3PATH=/you/frontier_client/python/lib:${PYTHON3PATH}
 and run either https://github.com/cms-sw/cmssw/blob/master/CondCore/Utilities/test/test_conddb.sh or just "conddb search 4b97f78682aac6254bbcba54cedbde468202bf5b" will also test the frontier client python binding.

To confirm the test, I removed `/cvmfs/cms.cern.ch/slc7_amd64_gcc10/external/frontier_client/2.9.1-1a7a47535e14b3c7ccfb7174b32f95b4/python/lib` from the `tool-deps.pth` file usually found in `/cvmfs/cms.cern.ch/slc7_amd64_gcc10/cms/cmssw-tool-conf/52.0-ddda3f06e541677ac005e786e7381a15/lib/python3.9/site-packages/` (listed in `PYTHON3PATH`), which ensured that the Frontier client code was coming from my new version and not from cvmfs.

### CODE REVIEW

**Comment by cvuosalo** [comment]
@DrDaveD Please review this request.

File: client/Makefile [hunk 1]
```diff
@@ -13,7 +13,7 @@
 #
 
 FN_VER_MAJOR	=	2
-FN_VER_MINOR	=	9.1
+FN_VER_MINOR	=	9.2
```
Comment (DrDaveD) [inline client/Makefile]: Let's make this version 2.10.1

File: client/RELEASE_NOTES [hunk 2]
```diff
@@ -1,3 +1,6 @@
+v2_9_2 - 7 Apr 2023 (cvuosalo)
+  - Add Python bindings and related patch that CMS has been using.
+
```
Comment (DrDaveD) [inline client/RELEASE_NOTES]: Don't have a blank line in here.

**Review by DrDaveD — CHANGES_REQUESTED** [review]
I don't think the frontier.py.rej and frontier.py.orig files should be included.

**Comment by cvuosalo** [comment]
@DrDaveD Your requested changes have been made.

**Review by DrDaveD — APPROVED** [review]

File: client/Makefile (hunk 1 above)
Comment (DrDaveD) [reply client/Makefile]: Oh I guess I should have asked for 2.10.0.  Oh well, we'll skip the .0 this time.

**Pull request merged**